src/transformers/hyperparameter_search.py

```diff
@@ -81,7 +83,13 @@ class RayTuneBackend(HyperParamSearchBackendBase):
 
     @staticmethod
     def is_available():
-        return is_ray_available()
+        tune_available = is_ray_tune_available()
+        if not tune_available:
+            return False
+
+        import ray
+
+        return tune_available and Version(ray.__version__) >= Version("2.7.0")
```

Comment:
This may not be the correct way to do this -- might need to raise an error to upgrade ray if someone tries to use this integration with an older verison